package.json

```diff
@@ -1,45 +1,51 @@
 {
   "name": "@byu-oit/logger",
-  "version": "0.3.14",
+  "version": "0.5.0",
```

Comment:
Looks like version 0.4.0 was never deployed 👀

Reply:
We could go back and fix the history, but it seems like extra effort at this point.

Reply:
The version jump bothered me, so I fixed it